app/controllers/idv/setup_errors_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Idv
+  class SetupErrorsController < ApplicationController
+    before_action :confirm_two_factor_authenticated
+
+    def show
```

Comment:
@zachmargolis Should we have an analytics event here?

Reply:
yeah that seems like a great idea! maybe a good follow-up PR if we want

and we might want to find a way to reuse `RedirectController` (maybe subclass it like we do for others) so we can tell how often users actually link the help center link?